dll/win32/msgina/shutdown.c

```diff
@@ -91,6 +93,37 @@ GetShutdownReasonUI(VOID)
 //    return (VersionInfo.wProductType == VER_NT_WORKSTATION) ? FALSE : TRUE;
 }
 
+static
+BOOL
+GetShutdownLunaUI(VOID)
+{
+    DWORD dwValue, dwSize;
+    HKEY hKey;
+    LONG lRet;
+
+    lRet = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
+                         L"Software\\Microsoft\\Windows\\CurrentVersion\\Reliability",
+                         0,
+                         KEY_QUERY_VALUE,
+                         &hKey);
+    if (lRet == ERROR_SUCCESS)
+    {
+        dwValue = 0;
+        dwSize = sizeof(dwValue);
+        RegQueryValueExW(hKey,
```

Comment:
Are these registry key/value names the same as Windows' ones?

Reply:
nope..

Reply:
Yes they are the same as in Windows Server 2003.

Reply:
See also:
https://www.remkoweijnen.nl/blog/2011/01/30/using-fast-user-switching-on-domain-xp-computers/

Reply:
@HBelusca I improved the checks, please see if everything is correct.